systers_portal/templates/common/about_us.html

```diff
@@ -0,0 +1,25 @@
+{% extends "base.html" %}
+{% load staticfiles %}
+
+{% block content %}
+  <div class="container col-md-8 col-md-offset-2">
+    <h1>About Systers</h1>
+    <img class="img-responsive" src="{% static 'img/about_systers.jpg' %}"
+           alt="Systers Communities">
+    <div class="mt40"></div>
+    <p class="lead">
```

Comment:
I am not sure if this `lead` class is perfect for long texts. How about trying to make it something similar to how blog entries look? We usually prefer to have Sans Serif titles and Serif fonts for body. You can create a new class that looks similar to `.blog-entry .body`.

Other than that, good job! Everything else is very good and thanks for remembering about tests.